src/axom/inlet/Table.cpp

```diff
@@ -72,24 +72,52 @@ Table& Table::addTable(const std::string& name, const std::string& description)
   return *currTable;
 }
 
-std::vector<std::pair<std::string, std::string>> Table::arrayIndicesWithPaths(
-  const std::string& name) const
+std::vector<std::string> Table::containerIndices() const
 {
-  std::vector<std::pair<std::string, std::string>> result;
-  if(!m_sidreGroup->hasView(ARRAY_INDICIES_VIEW_NAME))
+  std::vector<std::string> indices;
+  // If it's in the view, they are integer indices of an array
+  if(m_sidreGroup->hasView(CONTAINER_INDICES_NAME))
   {
-    SLIC_ERROR(fmt::format(
-      "[Inlet] Table '{0}' does not contain an array of user-defined objects",
-      m_name));
+    const auto view = m_sidreGroup->getView(CONTAINER_INDICES_NAME);
+    const int* array = view->getArray();
+    const auto num_elements = view->getNumElements();
+    indices.reserve(num_elements);
+    for(int i = 0; i < num_elements; i++)
+    {
+      indices.push_back(std::to_string(array[i]));
```

Comment:
I noticed that we store the array indices as ints (in Sidre) then convert them to strings (out of this function) then convert them back to ints (in get<> function).

Is there no way around this? Is it for keeping the templating clean?

Reply:
Follow up question -- would the code break if you had a table like the following:
```lua
foo = {
  [ 1 ] = "an int-indexed value",
  ['1'] = "a string-indexed value"
}
```

Reply:
I had initially left the indices as ints so they could be stored contiguously, but in retrospect I think that was a mistake given that performance isn't a big concern here.  Using identical storage for string and integer keys allowed a lot of duplicated code to be removed.